src/Form/test-cases/www_buzzfeed_com_signup.html

```diff
@@ -109,7 +109,7 @@ <h1 class="bold xs-mb1 xs-text-2">Sign Up To BuzzFeed</h1>
 form signature in host form: 10289224486994181929
 field frame token: 0685299093D39AE644081FF8F922F684
 form renderer id: 1
-field renderer id: 7" data-ddg-testresultelementid="547893" data-manual-scoring="password">
+field renderer id: 7" data-ddg-testresultelementid="547893" data-manual-scoring="username">
```

Comment:
❓ Just confirming it's expected that our manual scoring value can be at odds with the details in `autofill-information` (which say, incorrectly, password)?

Reply:
It's expected (though rare). That was added by Chrome during the comparison project. We left it for reference, but we shouldn't rely on it.